google/cloud/aiplatform/jobs.py

```diff
@@ -64,6 +64,7 @@
     gca_job_state.JobState.JOB_STATE_FAILED,
     gca_job_state.JobState.JOB_STATE_CANCELLED,
     gca_job_state.JobState.JOB_STATE_PAUSED,
+    gca_job_state.JobState.JOB_STATE_RUNNING,
```

Comment:
This seems like it will break usage in other jobs and isn't consistent with the intention of this variable.